coralproject/talk
# don't show profile tab if user is logged out

## What does this PR do?
This small patch removes the tab bar for anonymous users. There is no functionality available when the user is logged out, so removing the tab bar reduces clutter and potential confusion.

## How do I test this PR?
After applying the patch, observe that the tab bar is gone when logged out, but appears when logged in.
